webapp/src/components/sidebar/ModelLibrary.vue

```diff
@@ -210,11 +212,34 @@ function sortModels(a: ModelMetadata, b: ModelMetadata): number {
 }
 
 
-function deleteModelConfirm(modelID: string){
-  toaster.default(`Click here to confirm the model deletion`, {
-    onClick: () => deleteModel(modelID),
-    duration: 10000,
-  });
+async function deleteModelConfirm(modelID: string){
+  const modelInfo = models.value.find(([id]) => id === modelID);
```

Comment:
```suggestion
  const modelInfo = models.value.find(id => id === modelID);
```
I think you can unwrap the argument if you want